packages/payment-experience-api/src/lib/vismaPay.ts

```diff
@@ -31,6 +34,29 @@ export const createUserRedirectUrl = async ({
       redirectUrl = new URL(paymentReturnUrlConfiguration.configurationValue)
       redirectUrl.pathname = 'success'
       redirectUrl.searchParams.append('orderId', order.orderId)
+
+      try {
+        let payments = await getPaymentForOrder(order)
```

Comment:
This logic maybe could be moved to controller as its not dependent for vismapay?

Reply:
- [ ] Change receipt send logic to paymentNotifyController